README.md

```diff
@@ -21,7 +21,7 @@
 
 # Create and deploy a Swift Web Application using Kitura
 
-> We have similar patterns available for [Node.js](https://github.com/IBM/nodejs-web-app), [Java Spring](https://github.com/IBM/spring-web-app), [Go](https://github.com/IBM/go-web-app), Python [Django](https://github.com/IBM/django-web-app) & [Flask](https://github.com/IBM/flask-web-app), and [Java Liberty](https://github.com/IBM/java-liberty-web-app).
+> We have similar applications available for [Node.js](https://github.com/IBM/nodejs-web-app), [Java Spring](https://github.com/IBM/spring-web-app), [Go](https://github.com/IBM/go-web-app), Python [Django](https://github.com/IBM/django-web-app) & [Flask](https://github.com/IBM/flask-web-app), and [Java Liberty](https://github.com/IBM/java-liberty-web-app).
```

Comment:
for this section below:
 `bx dev deploy -t buildpack` or `bx dev deploy -t container` 

use `ibmcloud` instead and put those 2 commands on their own lines, similar to build/run